pkg/client/client.go

```diff
@@ -34,6 +34,7 @@ func NewCRIContainerdClient(endpoint string, timeout time.Duration) (api.CRICont
 		return nil, fmt.Errorf("failed to get dialer: %v", err)
 	}
 	conn, err := grpc.Dial(addr,
+		grpc.WithBlock(),
 		grpc.WithInsecure(),
 		grpc.WithTimeout(timeout),
```

Comment:
// WithTimeout returns a DialOption that configures a timeout for dialing a ClientConn initially. This is valid if and only if WithBlock() is present. Deprecated: use DialContext and context.WithTimeout instead.

Reply:
Note: we're adding something that's been deprecated

Reply:
Added TODO.